task-list/app/views/tasks/index.html.erb

```diff
@@ -18,6 +16,9 @@
         <th>
           Completed Date
         </th>
+        <th>
+
```

Comment:
If you're going to make an empty tag like this you can put both the opening and closing tags on the same line